### ISSUE TITLE
Unable to export user script. The page stuck forever with "Please wait..." message

### ISSUE THREAD

**Issue by saskk-aws** [Tampermonkey/tampermonkey#1042]
Not able to export any of the user scripts. 

## Expected Behavior
User scripts should be backed up.

## Actual Behavior
If I choose export option, the dialog box appears with following messages and roating forever.

**Please wait...**

## Specifications

- Chrome: (Version 85.0.4183.121 (Official Build) (64-bit))
- TM: (i.e. 4.11 2020-09-20)
- OS: (i.e. MacOS 10.15.7 (19H2))

<img width="1428" alt="Screenshot 2020-09-30 at 2 36 03 PM" src="https://user-images.githubusercontent.com/70074241/94665862-4ff68d80-032a-11eb-9fcf-20a4de266948.png">

**Comment by derjanb** [comment]
Hmm, I can not reproduce this. :-/ Do you see errors at the option (Ctrl+Shift+J) oder background ([FAQ](https://www.tampermonkey.net/faq#Q600)) page's console?

**Comment by saskk-aws** [comment]
Following appeared even before trying to export.
```
registry.js:1 registry: finished loading layout/default/layout_helper.js 
registry.js:1 registry: finished loading layout.js 
statistics.js:5 statistics: Unable to load GA -> enable emergency mode
codemirror.js:207 [Violation] Added non-passive event listener to a scroll-blocking 'touchstart' event. Consider marking event handler as 'passive' to make the page more responsive. See https://www.chromestatus.com/feature/5745543795965952
w @ codemirror.js:207
kh @ codemirror.js:181
E @ codemirror.js:179
window.MirrorFrame @ mirrorframe.js:2
(anonymous) @ options.js:144
e.check @ promise.js:1
e.push @ promise.js:1
done @ promise.js:1
onShow @ options.js:144
(anonymous) @ options.js:90
g @ helper.js:1
onShow @ options.js:90
Cb.v.<computed>.scriptClick @ options.js:151
(anonymous) @ options.js:162
Ha @ options.js:31
(anonymous) @ options.js:18
codemirror.js:207 [Violation] Added non-passive event listener to a scroll-blocking 'touchmove' event. Consider marking event handler as 'passive' to make the page more responsive. See https://www.chromestatus.com/feature/5745543795965952
w @ codemirror.js:207
kh @ codemirror.js:182
E @ codemirror.js:179
window.MirrorFrame @ mirrorframe.js:2
(anonymous) @ options.js:144
e.check @ promise.js:1
e.push @ promise.js:1
done @ promise.js:1
onShow @ options.js:144
(anonymous) @ options.js:90
g @ helper.js:1
onShow @ options.js:90
Cb.v.<computed>.scriptClick @ options.js:151
(anonymous) @ options.js:162
Ha @ options.js:31
(anonymous) @ options.js:18
codemirror.js:207 [Violation] Added non-passive event listener to a scroll-blocking 'mousewheel' event. Consider marking event handler as 'passive' to make the page more responsive. See https://www.chromestatus.com/feature/5745543795965952
w @ codemirror.js:207
kh @ codemirror.js:183
E @ codemirror.js:179
window.MirrorFrame @ mirrorframe.js:2
(anonymous) @ options.js:144
e.check @ promise.js:1
e.push @ promise.js:1
done @ promise.js:1
onShow @ options.js:144
(anonymous) @ options.js:90
g @ helper.js:1
onShow @ options.js:90
Cb.v.<computed>.scriptClick @ options.js:151
(anonymous) @ options.js:162
Ha @ options.js:31
(anonymous) @ options.js:18
[Violation] Forced reflow while executing JavaScript took 58ms
```

**Comment by derjanb** [comment]
Unfortunately none of these messages is export related. Are there more messages? Have you checked the background page? Thanks.

**Comment by saskk-aws** [comment]
No more messages apart from that.
Could you help me checking background page? What does that mean ?

**Comment by derjanb** [comment]
> background ([FAQ](https://www.tampermonkey.net/faq#Q600)) page's console?

Please check Q600 at the FAQ.

**Comment by saskk-aws** [comment]
Thanks. I did not know about that debugging.

Following are the error printed as soon as I clicked on export button.
```
background.html:1 Error in event handler: TypeError: Message length exceeded maximum allowed length.
    at chrome-extension://dhdgffkkebhmkfjojejmpbldmpobfkfo/background.js:181:405
    at Object.<anonymous> (chrome-extension://dhdgffkkebhmkfjojejmpbldmpobfkfo/background.js:170:21)
    at Array.e.check (chrome-extension://dhdgffkkebhmkfjojejmpbldmpobfkfo/promise.js:1:296)
    at Array.e.push (chrome-extension://dhdgffkkebhmkfjojejmpbldmpobfkfo/promise.js:1:162)
    at Object.done (chrome-extension://dhdgffkkebhmkfjojejmpbldmpobfkfo/promise.js:1:371)
    at Object.exec (chrome-extension://dhdgffkkebhmkfjojejmpbldmpobfkfo/background.js:170:12)
    at f (chrome-extension://dhdgffkkebhmkfjojejmpbldmpobfkfo/background.js:181:387)
```

FYI, I am using 67 user-scripts in total.

**Comment by derjanb** [comment]
> TM: (i.e. 4.11 2020-09-20)

Wahh! I just saw, that your screenshot shows TM version 4.10 while the description mentioned 4.11. The issue should be fixed in 4.11 

In case Chrome has not updated your Tampermonkey to 4.11 yet, then please download it from here:
https://tampermonkey.net/crx/tampermonkey_4_11.crx

and drag'n'drop it to the `chrome://extensions` page.
